.github/workflows/labeler-triage.yml

```diff
@@ -1,4 +1,9 @@
 name: "Pull Request Labeler"
+
+permissions:
+  contents: read
+  pull-requests: write
```

Comment:
do these govern what scopes the `secrets.GITHUB_TOKEN` has?

Reply:
Yep, that's right. The current default gives access to a lot more, but I believe it is now best practice to explicitly grant exactly what is required.

Reply:
This looks great ❤️